hydra-derivatives.gemspec

```diff
@@ -17,16 +17,13 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 2.1'
   spec.add_development_dependency 'fcrepo_wrapper', '~> 0.2'
   spec.add_development_dependency 'pry-byebug'
-  spec.add_development_dependency 'rails', '> 5.1', '< 7.0'
-  spec.add_development_dependency 'rake', '~> 10.1'
+  spec.add_development_dependency 'rails', '> 5.1', '< 7.1'
+  spec.add_development_dependency 'rake'
   spec.add_development_dependency 'rspec', '~> 3.1'
-  spec.add_development_dependency "solr_wrapper", "~> 2.0"
+  spec.add_development_dependency "solr_wrapper"
 
-  spec.add_dependency 'active-fedora', '>= 11.5.6',
-                      '!= 12.0.0', '!= 12.0.1', '!= 12.0.2', '!= 12.0.3', '!= 12.1.0', '!= 12.1.1', '!= 12.2.0', '!= 12.2.1',
-                      '!= 13.0.0', '!= 13.1.0', '!= 13.1.1', '!= 13.1.2', '!= 13.1.3', '!= 13.2.0', '!= 13.2.1'
```

Comment:
Did you mean to remove this?  I think it might still needed...at least the `active-fedora` dependency is needed.